public/js/map.js

```diff
@@ -1,27 +1,28 @@
+import 'whatwg-fetch';
+
+const L = require('mapbox.js');
```

Comment:
Any idea why it's the norm with mapbox to save the import to a var named `L`?

Reply:
Heh - yeah I think it's probably from mapbox's origin as an extension to Leaflet JS. 🤷‍♂